snippets/app-router/layout.mdx

```diff
@@ -3,6 +3,7 @@ import type { Metadata } from "next";
 import { Inter } from "next/font/google";
 import { MakeswiftProvider } from "@/makeswift/provider";
 import { DraftModeScript } from "@makeswift/runtime/next/server";
+import "@/makeswift/components";
```

Comment:
I wonder if we should utilize some sort of poor man's code callouts here and below to bring attention to the three things that we are listing in the paragraph that precedes this example:

<img width="753" alt="para" src="https://github.com/user-attachments/assets/5a8767a6-3555-44f2-a2fb-11ac6b5c9725">

E.g.

```typescript
import "@/makeswift/components"; // import the registered components
```

Or maybe it's possible to integrate something like [CodeHike](https://codehike.org/) into our docs to get https://codehike.org/docs/code/footnotes?

Reply:
Unfortunately Mintlify has little customizability. We're planning on moving to a custom docs build so that we can do stuff like that as much as we want.

Reply:
We can do "callouts" through comments, though, right?

Reply:
Yeah! 👍🏻 

There might also be a feature in Mintlify that gets close to what we need